src/Cache/JsonCacheKeyGenerator.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace ScriptFUSION\Porter\Cache;
+
+class JsonCacheKeyGenerator implements CacheKeyGenerator
+{
+    public function generateCacheKey($source, array $optionsSorted)
+    {
+        return str_replace(str_split('{}()/\@:'), '.', json_encode([$source, $optionsSorted], JSON_UNESCAPED_SLASHES));
```

Comment:
Please make sure all occurrences are replaced with the constant. I think there is also a test I wrote yesterday that could be updated.